src/main/java/org/spongepowered/api/text/channel/ChatTypeMessageReceiver.java

```diff
@@ -30,14 +30,29 @@
 import org.spongepowered.api.text.TextElement;
 import org.spongepowered.api.text.TextTemplate;
 import org.spongepowered.api.text.chat.ChatType;
+import org.spongepowered.api.text.chat.ChatTypes;
 
 import java.util.Map;
 
 /**
- * Represents something that can receive messages of certain types.
+ * Represents a {@link MessageReceiver} that can receive messages with a
+ * specified {@link ChatType}, as well as send messages through the use
+ * of a {@link MessageChannel}.
  */
-@FunctionalInterface
-public interface ChatTypeMessageReceiver {
+public interface ChatTypeMessageReceiver extends MessageReceiver {
+
+    /**
+     * Sends a message as a {@link ChatTypes#CHAT} message.
+     *
+     * <p>If text formatting is not supported in the implementation
+     * it will be displayed as plain text.</p>
+     *
+     * @param message The message to send
+     */
+    @Override
+    default void sendMessage(Text message) {
+        sendMessage(ChatTypes.CHAT, message);
```

Comment:
Shouldn't the default `ChatType` be `ChatTypes.SYSTEM`, since most plugins' messages sent to players aren't really chat?

Reply:
[Just following the conversation here](https://github.com/SpongePowered/SpongeAPI/pull/2004#discussion_r285146287) but I see your point.

@Zidane - thoughts?

Reply:
Yes, it should be `SYSTEM`.

Reply:
Well I could be wrong for once :p.

Reply:
Yeah, I just blindly did it to get out of the way. It'll get changed.